subsys/bluetooth/audio/bap_scan_delegator.c

```diff
@@ -245,13 +245,13 @@ static void receive_state_updated(struct bt_conn *conn,
 	bt_debug_dump_recv_state(internal_state);
 	net_buf_put_recv_state(internal_state);
 	bass_notify_receive_state(internal_state);
+	k_sem_give(&read_buf_sem);
```

Comment:
`bass_notify_receive_state` uses the `read_buf`, but after this returns the `read_buf` can be used by others